README.md

```diff
@@ -284,6 +284,7 @@ You can customize your deployment with env variables. See .env.example for all p
 
 #### Capacity Related
 
+- UPDATE_POOLS_INTERVAL_IN_MINUTES - default: 1 - How frequently to run the update pools lambda.
```

Comment:
I think it should default to 5 minutes as in dev accounts you don't need such frequent updates.